test/position-managers/FeeCollection.t.sol

```diff
@@ -68,6 +68,7 @@ contract FeeCollectionTest is Test, PosmTestSetup, LiquidityFuzzers {
         uint256 balance1Before = currency1.balanceOfSelf();
         BalanceDelta delta = collect(tokenId, ZERO_BYTES);
 
+        assertApproxEqAbs(uint256(int256(delta.amount1())), swapAmount.mulWadDown(FEE_WAD), 1 wei);
```

Comment:
instead of using swapAmount.mulWadDown can you use getFeesOwed? I think its more precise.. and then try to do an exact assertion?

Reply:
we're asserting delta == feesOwed in the line below, so gonna keep this line as-is since its a nice assertion that feesOwed ~= swapAmount * FEE_WAD